autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/LoadAlertConfigurationRequest.java

```diff
@@ -17,14 +17,14 @@
 public class LoadAlertConfigurationRequest implements Json {
     @ApiModelProperty(LoadAlertJsonProperties.LOAD_ALERT_CONFIGURATION_MIN_RESOUCE_VALUE)
     @Min(value = 0)
-    @Max(value = 200)
+    @Max(value = 500)
     @Digits(fraction = 0, integer = 3)
     @NotNull
```

Comment:
CB now supports manual scaling up-to 500 also, hence updating autoscaling limits also.